apps/i18n/music/en_us.json

```diff
@@ -52,6 +52,7 @@
   "blockly_toolboxCategorySimple": "Simple",
   "blockly_toolboxCategoryTracks": "Tracks",
   "blockly_toolboxCategoryVariables": "Variables",
+  "buildWithMusicLab": "Build with Music Lab",
```

Comment:
was this supposed to be `Built`? (jw based on slack convo this morning)

Reply:
🙏  - if I could, I'd give you a very big hug right now.